lib/hawkular/base_client.rb

```diff
@@ -175,6 +178,16 @@ def tenant_header
       headers
     end
 
+    # @!visibility private
```

Comment:
could you explain what is this comment/attribute? 
(btw, based on line 169 this is already part of a private section)